test/hazelcast_json_value/HJVJsonSerializerTest.js

```diff
@@ -130,11 +107,73 @@ describe('HazelcastJsonValue with JsonSerializer', function () {
             expect(value).to.deep.equal(object);
         });
     });
+});
+
+describe('Querying with HazelcastJsonValue', function () {
```

Comment:
This is my personal opinion but having multiple describe blocks in a single test file makes it harder to find failed tests in PR builder and nightly runs. I would prefer each describe block have its own ...Test.js file with a similar name to its label.

Reply:
Separated them